Sources/Apollo/ApolloClient.swift

```diff
@@ -1,22 +1,34 @@
+import ApolloCore
 import Foundation
 import Dispatch
 
 /// A cache policy that specifies whether results should be fetched from the server or loaded from the local cache.
-public enum CachePolicy {
+public enum CachePolicy: Equatable {
```

Comment:
Why does `CachePolicy` need to conform to `Equatable`?

Reply:
Doesn't need to, but it doesn't hurt to have it - want me to revert this as well?

Reply:
If it does not need to I would not add that conformance.